composer.json

```diff
@@ -12,9 +12,7 @@
     "require": {
         "php": ">=7.4.0",
         "ext-fileinfo": "*",
-        "illuminate/support": "^8.83.3|^9.0",
-        "illuminate/filesystem": "^8.83.3|^9.0",
-        "illuminate/database": "^8.83.3|^9.0",
+        "laravel/framework": "^9.0 | ^10.0",
```

Comment:
can you keep 8.X for now? dropping support is a major version change. That should be done, but there are other requests to be included in such a version, so I would like to take some additional time to plan it out.

I believe the testbench dependency in require-dev also needs a version bump for compatibility